plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -350,7 +353,26 @@ class CodeWhispererCodeScanManager(val project: Project) {
                     "stacktrace: ${e.stackTrace.contentDeepToString()}"
             }
         }
-        return errorMessage
+
+        val telemetryErrorMessage = when (e) {
+            is CodeWhispererException -> e.awsErrorDetails().errorMessage() ?: message("codewhisperer.codescan.run_scan_error_telemetry")
+            is CodeWhispererCodeScanException -> when (e.message) {
+                message("codewhisperer.codescan.no_file_open") -> message("codewhisperer.codescan.no_file_open_telemetry")
```

Comment:
I thought we are not adding any no file open errors.

Reply:
IDK the timeline of which PR merges first so I thought it will be safe to add it and remove this if required in next release. As this will not disturb the end customer(just a safety check) if there is no occurrence of this exception.

Reply:
Makes sense...you can leave it in.